releases/index.rst

```diff
@@ -11,11 +11,11 @@ Borg 1.2
 Borg 1.2 is the current **development** branch of Borg. There is no release in this series
 at this time.
 
-- `Wiki page <https://github.com/borgbackup/borg/wiki/Borg-1.2>`__
+- `Wiki page <https://github.com/borgbackup/borg/wiki/Borg-1.2-Hydrogen>`__
 - `Project board <https://github.com/borgbackup/borg/projects/4>`__
 - `Documentation <https://borgbackup.readthedocs.io/en/latest/>`__
 - `Installation <https://borgbackup.readthedocs.io/en/latest/installation.html>`__
-- `Downloads <https://github.com/borgbackup/borg/releases/>`__
+.. - `Downloads <https://github.com/borgbackup/borg/releases>`__
```

Comment:
is the .. an accidental change?

Reply:
No, I commented it out because there are no downloads for 1.2 yet.

Reply:
ah, right. just kill the line then? can be easily added again once we have downloads.